dockers/docker-fpm-frr/supervisord.conf

```diff
@@ -35,6 +35,8 @@ autorestart=unexpected
 startsecs=0
 stdout_logfile=syslog
 stderr_logfile=syslog
+dependent_startup=true
+dependent_startup_wait_for=start.sh:exited
```

Comment:
>start.sh:exited [](start = 27, length = 15)

Will rsyslogd start after start.sh finish? If yes, we miss some initial logs.

Reply:
yes, but it is existing logic.  I do not plan to address this in this pr.

Reply:
Existing logic is not perfect, but here is worse.

---
In reply to: [425515652](https://github.com/Azure/sonic-buildimage/pull/4599#discussion_r425515652) [](ancestors = 425515652)

Reply:
explian more?

Reply:
It will miss some logs immediately after
```
supervisorctl start bgp_eoiu_marker
```

---
In reply to: [425517528](https://github.com/Azure/sonic-buildimage/pull/4599#discussion_r425517528) [](ancestors = 425517528,425515652)